Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -26,10 +26,14 @@ configuration:
   name: incidents_per_fetch
   required: false
   type: 0
-- display: Fetch query
+- display: Detections fetch query
   name: fetch_query
   required: false
   type: 0
+- display: Incidents fetch query
+  name: incidents_fetch_query
```

Comment:
@roysagi Am I missing something? looks like he added a new param, which isn't a BC break. Would love to understand what you meant to say

Reply:
@guykeller - If I am getting this right, till now if a user wanted to fetch incidents (not detections) from CS - he used the `fetch_query` param - am I getting this wrong @ChanochShayner?

If so, this means that now that param is only relevant for detections and anyone that fetches incidents currently with this param will have a BC break.

Reply:
@roysagi You're right, but there seems to be no way to do it without BC break

Reply:
It can be carefully assumed that no one used it for incidents because the fetch incidents with a bug and more people would have reported it.
@guykeller What is the decision?